client/question/components/LearnerViewDirective.js

```diff
@@ -89,14 +89,19 @@ tie.directive('learnerView', [function() {
                     </ui-codemirror>
                   </div>
                 </div>
-                <select ng-if="SERVER_URL"
-                    class="tie-select-menu"
-                    name="lang-select-menu">
-                  <option value="Python" selected>Python</option>
-                </select>
                 <button class="tie-code-reset tie-button protractor-test-reset-code-btn" name="code-reset" ng-click="resetCode()" title="Click to clear your code">
                   Reset Code
                 </button>
+                <p class="tie-language-label">Language: {{supportedLanguageCount == 1 ? languageLabel : ""}}</p>
```

Comment:
Could you do: `<p ng-if="supportedLanguageCount === 1">` instead (and remove the ternary expression)? I think that would be cleaner.

Reply:
Updated to remove the ternary expression. Intentionally left the "Language:", as a label for the dropdown as needed.